independent-publisher-connectors/Cohere/apiDefinition.swagger.json

```diff
@@ -0,0 +1,847 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cohere",
+    "description": "Cohere is making natural language processing part of every developer's toolkit. Harness the power of language understanding. Join the developers and businesses who are using Cohere to generate, categorize and organize text at a scale that was previously unimaginable.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.cohere.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/generate": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
@troystaylor is it possible to have more proper description for the responses. you can use something like "OK", "Success", "Bad Request", "200". This will help investigating issues in the future :-).
Please update all responses' descriptions in the swagger file.